k8s_services_srv.py

```diff
@@ -208,17 +175,88 @@ def get_k8s_endpoint_node(name, namespace):
             return ""
 
 
+def diff(listA, listB):
+    return [i for i in listA if i not in listB]
```

Comment:
return set(listA) - set(listB)

Reply:
Non, ça marche pas car c'est une liste de Dict : 
`Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'dict'`